input/fsh/MedComDocumentSourcePatient.fsh

```diff
@@ -3,6 +3,7 @@ Parent: DkCorePatient
 Id: medcom-document-patient
 Description: "A profile including requirements for a MedCom Document Patient."
 * text MS
+* identifier 1..1
```

Comment:
Er det for læsbarheden, at du har tilføjet den, eller skal den også stå uden MS først?